Updated image and JavaScript function and created new sections

gracieorozco opened the pull request:
I updated the image to be a picture of myself and the function to reflect a fact on me. I also created new sections, they are: background, projects, LinkedIn and Github, image gallery, what I like to photograph, and hidden talent blog posts. Afterwards, I deployed the project to a live server.

KaylaNguyen commented on portfolio/src/main/webapp/index.html:
```diff
@@ -8,11 +8,224 @@
+          the following day. 
+      </p>
       <div id="greeting-container"></div>
+      <p>Click here to get a random message:</p>
```
You probably want to switch order of these 2 components so the random message shows up after "Click here to get...."

KaylaNguyen commented on portfolio/src/main/webapp/script.js:
```diff
@@ -16,13 +16,6 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
```
Should I add more greetings so this function actually returns a "random" one?

KaylaNguyen commented on portfolio/src/main/webapp/index.html:
```diff
@@ -8,11 +8,224 @@
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Gracie's Portfolio</h1>
```
Nice job on the portfolio! =)

gracieorozco commented:
I have fixed the adjustments you mentioned above by changing the placement of the div and adjusted the function name and variables to reflect what it does.

KaylaNguyen commented on portfolio/src/main/webapp/script.js:
```diff
@@ -13,16 +13,18 @@
+  // Pick a random fact.	
+  const fact = facts[Math.floor(Math.random() * facts.length)];
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = "\n" + fact;
```
You may not need this. If you want a space between "Click here.." and the message/fact, you should add this to the index.html page instead.
